src/ajax.js

```diff
@@ -75,6 +75,7 @@ export function ajaxBuilder(timeout = 3000, {request, done} = {}) {
       if (options.withCredentials) {
         x.withCredentials = true;
       }
+      x.setRequestHeader('X-Prebid', 'pbjs/$prebid.version$');
```

Comment:
Sorry for the change here @snapwich , but the PBS committee today noted that the prefered HTTP header syntax is lower case. May I ask you to change this to "x-prebid" please?

Reply:
i'll change the `X-Requested-With` below it to be lowercase as well just to be consistent.